cmd/syft/cli/options/fulcio.go

```diff
@@ -0,0 +1,48 @@
+package options
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+const DefaultFulcioURL = "https://fulcio.sigstore.dev"
+
+// FulcioOptions is the wrapper for Fulcio related options.
+type FulcioOptions struct {
+	URL                      string
+	IdentityToken            string
+	InsecureSkipFulcioVerify bool
+}
+
+var _ Interface = (*FulcioOptions)(nil)
+
+// AddFlags implements Interface
+func (o *FulcioOptions) AddFlags(cmd *cobra.Command, v *viper.Viper) error {
+	// TODO: change this back to api.SigstorePublicServerURL after the v1 migration is complete.
+	cmd.Flags().StringVar(&o.URL, "fulcio-url", DefaultFulcioURL,
+		"address of sigstore PKI server")
+
+	cmd.Flags().StringVar(&o.IdentityToken, "identity-token", "",
+		"identity token to use for certificate from fulcio")
+
+	cmd.Flags().BoolVar(&o.InsecureSkipFulcioVerify, "insecure-skip-verify", false,
+		"skip verifying fulcio published to the SCT (this should only be used for testing).")
```

Comment:
What's the SCT? The docs should probably reflect the full name.

Reply:
Also I used SCT incorrectly here so double good call out.

SCT is “Signed Certificate Timestamp”. We need this flag so that we don't have to setup a root CA for fulcio during testing.  Fulcio returns a “Signed Certificate Timestamp”.

See here if you want the deep dive:
https://blog.chainguard.dev/a-fulcio-deep-dive/